package.json

```diff
@@ -11,7 +11,7 @@
         "clean:ts": "shx rm -rf lib",
         "clean:docker": "shx rm -rf 0x_mesh/db",
         "build": "tsc -p tsconfig.json",
-        "test": "ETHEREUM_RPC_URL=http://localhost:8545 CHAIN_ID=1337 RFQT_API_KEY_WHITELIST='koolApiKey1,koolApikey2' RFQT_MAKER_ENDPOINTS='https://mock-rfqt1.club' mocha --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js --timeout 100000 --exit",
+        "test": "ETHEREUM_RPC_URL=http://localhost:8545 CHAIN_ID=1337 RFQT_API_KEY_WHITELIST='koolApiKey1,koolApikey2' RFQT_MAKER_ENDPOINTS='https://mock-rfqt1.club' mocha --require source-map-support/register --require make-promises-safe lib/test/**/*_test.js --timeout 200000 --exit",
```

Comment:
Are we defining the test env here or in that object in the test file?

Reply:
We're defining it in that object in the test file. I'll remove this today

Reply:
This actually requires a bigger refactor within the API for this to work. We still need to add some of the custom envvars to the `docker-compose.yml` though. For example, the `ETHEREUM_RPC_URL` needs to be `http://ganache:8545` for `docker-compose up` to work correctly.

Reply:
I think it would be worth it to get the `CHAIN_ID` from the `provider` if that wouldn't be too much work. It seems somewhat out of scope for this PR though.